doc/content/getting-started/migrating-from-v2/major-changes.md

```diff
@@ -0,0 +1,82 @@
+
+<!--
+TODO: https://github.com/TheThingsNetwork/lorawan-stack/issues/2421
+Add reference to docs after that is merged.
+-->
+
+Furthermore, {{% tts %}} brings full support for all LoRaWAN versions, as well as Class B and Class C end devices.
+
+## Gateway Coverage
+
+The Packet Broker enables peering between networks, so traffic received by one network (e.g. the Public Community Network) but intended for a different network ({{% tts %}}) can be forwarded to and from that network. See the [Peering Guide]({{< ref "/integrations/peering" >}}) for details on Packet Broker and how to enable it for your network.
+
+With Packet Broker enabled on both {{% tts %}} and {{% ttnv2 %}}, you can receive traffic on {{% tts %}} without having to re-configure any of your gateways.
+
+> **NOTE**: Packet Broker is enabled on The Things Network Public Community Network.
+
+For private {{% tts %}} deployments with Packet Broker disabled, you will need to re-configure your gateways to connect to {{% tts %}}, so that you can start receiving traffic from your end devices.
+
+In order to connect a gateway to {{% tts %}}, follow instructions for [Adding a Gateway in the Console]({{< ref "/getting-started/console/create-gateway" >}}) or [Adding a Gateway using the CLI]({{< ref "/getting-started/cli#create-gateway" >}}). Then, reconfigure the gateway to connect to {{% tts %}}, and regenerate its API key (if required).
+
+Also see [Gateways]({{< ref "/gateways" >}}) for instructions on configuring popular LoRaWAN gateways with {{% tts %}}.
+
+## Application Data
+
+{{% tts %}} uses a different data format for uplink and downlink traffic than {{% ttnv2 %}}.
+
+For details on the data format of {{% ttnv2 %}}, see the documentation from [The Things Network](https://www.thethingsnetwork.org/docs/applications/mqtt/api.html).
+
+The data format has changed in {{% tts %}}. It uses a different schema, different names, and has much richer metadata support. Read more about that in [Data Formats]({{% ref "/integrations/data-formats" %}}).
+
+When migrating to {{% tts %}}, ensure your application can properly handle the new {{% tts %}} data format.
+
+### Payload Formats
+
+{{% ttnv2 %}} has support for payload decoders, converters, validators (for uplink) and encoders (for downlink). These can be either CayenneLPP or Javascript functions.
+
+{{% tts %}} has support for an uplink payload formatter (similar to the payload decoder) and a downlink payload formatter (similar to the payload encoder). These can be set per application, and can even be overridden per end device. Similar to {{% ttnv2 %}}, CayenneLPP and Javascript functions are supported.
+
+Migrating the {{% ttnv2 %}} payload encoder and decoder to an uplink and downlink payload formatter should be straightforward, since they have the same format.
+
+<!--
+TODO: https://github.com/TheThingsNetwork/lorawan-stack/issues/598
+Add a reference here once Payload Formatters are documented.
+-->
+
+### MQTT Traffic
```

Comment:
MQTT is no different than webhooks; please add a section to that as well

The payload format changed as well as downlink endpoints, paths and security settings. Also, it's now called Webhooks (it was HTTP integration)